examples/js/loaders/GLTFLoader.js

```diff
@@ -1949,156 +1926,177 @@ THREE.GLTFLoader = ( function () {
 
 				if ( primitive.indices !== undefined ) {
 
-					geometry.setIndex( dependencies.accessors[ primitive.indices ] );
+					geometry.setIndex( accessors[ primitive.indices ] );
 
 				}
 
-				return geometry;
+				geometries.push( geometry );
 
-			} );
+			}
+
+			return geometries;
 
 		} );
 
 	};
 
 	/**
 	 * Specification: https://github.com/KhronosGroup/glTF/blob/master/specification/2.0/README.md#meshes
+	 * @param {number} meshIndex
+	 * @return {Promise<THREE.Mesh|THREE.SkinnedMesh>}
```

Comment:
Should the return type be `Promise<THREE.Group>`?

Reply:
Oh, it should actually be:

```
Promise<THREE.Mesh|THREE.SkinnedMesh|THREE.Group>
```

in the future we want to reduce the number of cases where a Group is returned (#11944), but probably can't eliminate that entirely.

Reply:
Oh, I see you spotted that already 😁

Reply:
:)